pkg/provisioner/ironic/ironic.go

```diff
@@ -393,6 +422,12 @@ func (p *ironicProvisioner) ValidateManagementAccess(credentialsChanged, force b
 		return
 	}
 
+	validationError = bmcAccess.Validate(p.bmcCreds, p.bmcClient)
+	if validationError != nil {
```

Comment:
I think you need to check for AccessVirtualMediaValidationError and only log a warning in this case.